### DISCUSSION TITLE
Is this being actively developed and maintained?

### DISCUSSION THREAD

**Discussion by heath-freenome** [FormidableLabs/victory#2576] [General]
I'm seeing very few recent commits and people asking how they can help get features over the line (i.e. heatmap) with no responses in almost a year. So, I have to wonder whether this project is still alive?

**Comment by carloskelly13** [comment]
Victory is an active project for us and our community. We've had an involved effort throughout the latter part of last year to migrate a big chunk of the core to TypeScript.

Our open source work is accomplished through a combination of need for our related client work, folks in between client projects, and passionate interest outside of work hours (that we never want to depend on outside individual interest). Victory is one part of our priorities in the portfolio, and we do keep a keen eye on issues/contributions that we can move forward when we have the bandwidth to immediately work on it. We expect as our folks rotate on and off projects to have cadence on Victory ramp up again in the future (and then assuredly down some time later by the same token).

In addition to core feature and maintenance work, we've invested a lot of time and effort over the last quarter to significantly refactor and enhance the underlying infrastructure and DX to make development much more tractable for the community. As a large, multi-component library, Victory relies on community support and help to advance the library alongside us. We hope that you and the community find this much easier now.

Outside of our between project availability and community contributions, we do occasionally have folks that have need and budget to get specific Victory features/behavior shipped -- we always love to help in these situations. Just reach out!
